client/src/components/RepositoryHeader/index.jsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
+import { Link, withRouter } from 'react-router-dom';
 import Octicon, { Repo } from '@primer/octicons-react';
 import { Icon, Label, Container } from 'semantic-ui-react';
 import ForkButton from '../ForkButton';
 
 import styles from './styles.module.scss';
 
-const RepositoryHeader = ({ owner, username, repoName, forkCount, issueCount, activePage, baseUrl }) => {
+const goToRootDir = (history, url) => () => {
+  history.push(url);
+  window.location.reload();
```

Comment:
Seems tricky. Are you sure about that?

Reply:
Do you mean closure? 
It would be easier to use arrow function, but eslint dont allow to use it in props

Reply:
No, I mean `window.location.reload();`